src/main/java/frc/robot/RobotContainer.java

```diff
@@ -51,6 +62,17 @@ public RobotContainer() {
   private void configureBindings() {
     new Trigger(() -> triggerPressed())
         .whileTrue(new BasicDriveCommand(m_robotDrive, m_driverController));
+
+    // TODO: make actually intake under button press
+    new Trigger(this::buttonPressed)
+        .whileTrue(new RunIntake(m_intake));
+  }
+
+  public boolean buttonPressed() {
```

Comment:
Which button? The controller has many.

Reply:
Responded to

Reply:
I mean that the function should be self-descriptive. Javadoc your functions as well.